src/textual/widgets/_directory_tree.py

```diff
@@ -116,6 +118,7 @@ def __init__(
             classes: A space-separated list of classes, or None for no classes.
             disabled: Whether the directory tree is disabled or not.
         """
+        self._to_load: Queue[TreeNode[DirEntry]] = Queue()
```

Comment:
I don't find this name to be descriptive. Maybe `_load_queue` ?

Reply:
Makes sense.